GoogleSignIn/Sources/GIDSignIn.m

```diff
@@ -652,9 +659,14 @@ - (void)authorizationRequestWithOptions:(GIDSignInInternalOptions *)options comp
         if (token) {
           additionalParameters[kClientAssertionTypeParameter] = kClientAssertionTypeParameterValue;
           additionalParameters[kClientAssertionParameter] = token.token;
-          request = [self authorizationRequestWithOptions:options
-                                     additionalParameters:additionalParameters];
         }
+        #if DEBUG
+        if (error) {
+          NSLog(@"[Google Sign-In iOS]: Error retrieving App Check limited use token: %@", error);
+        }
+        #endif
```

Comment:
Nice, I think this is a good compromise vs. entirely swallowing the error. In a follow-up, you could consider using [OSLog](https://developer.apple.com/documentation/os/logging/generating_log_messages_from_your_code?language=objc) like:
```
os_log_t customLog = os_log_create("com.google.sign-in", "App Check");
os_log_with_type(customLog, OS_LOG_TYPE_ERROR, Error retrieving App Check limited use token: %@", error);
```

Xcode 15 makes it easy to customize which logging levels, categories, etc. are shown if you use OSLog.

Reply:
Thanks for the `OSLog` recommendation. I've created a tracking issue for this.

https://github.com/google/GoogleSignIn-iOS/issues/338